src/renderers/dom/shared/ReactDOMComponentTree.js

```diff
@@ -95,7 +92,8 @@ function precacheChildNodes(inst, node) {
   }
   var children = inst._renderedChildren;
   var childNode = node.firstChild;
-  outer: for (var name in children) {
+  outer:
```

Comment:
This is a leftover from the recast fork. I'll remove the \n, it doesn't make sense.